src/aap_eda/core/migrations/0021_rulebookprocess_event_stream_and_more.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.7 on 2024-02-07 14:13
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import aap_eda.core.enums
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0020_eventstream"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="rulebookprocess",
+            name="event_stream",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="event_stream_processes",
+                to="core.eventstream",
+            ),
+        ),
+        migrations.AddField(
+            model_name="rulebookprocess",
+            name="parent_type",
+            field=models.TextField(
+                choices=[
+                    ("activation", "activation"),
+                    ("event_stream", "event_stream"),
+                ],
+                default=aap_eda.core.enums.ProcessParentType["ACTIVATION"],
```

Comment:
@Alex-Izquierdo does this take care of existing rows or is it just using a default.

Reply:
The field is mandatory, so the default is applied to all records. Since the activation was previously mandatory, we don't need any other action in the migration.

Reply:
ok